src/Native/Unix/System.Security.Cryptography.Native/pal_asn1.cpp

```diff
@@ -42,7 +42,7 @@ extern "C" int32_t CryptoNative_ObjSn2Nid(const char* sn)
     return OBJ_sn2nid(sn);
 }
 
-extern "C" ASN1_OBJECT* CryptoNative_ObjNid2Obj(int32_t nid)
+extern "C" const ASN1_OBJECT* CryptoNative_ObjNid2Obj(int32_t nid)
```

Comment:
This isn't a change in the ABI, right? Is it safe to do?

Reply:
That matches my understanding, yeah. :smile: